src/components/StopwatchEntry.vue

```diff
@@ -54,6 +54,14 @@ export default {
     stop: async function() {
       this.event.data.running = false;
       this.event.duration = (moment() - moment(this.event.timestamp)) / 1000;
+      // Convert the string to a list of tags
+      if(this.event.data.tags){
+        this.event.data.tags = this.event.data.tags.replace(/ /g,'')
+        this.event.data.tags = this.event.data.tags.split(',')
```

Comment:
This only allows for a single word in the tag, multi-word tags will be destroyed.
Would be better if it finds a comma it should then removes all types of spaces before it and after it.